### PR TITLE
Create announcing-sigstore-java-1-0.md

### PR DESCRIPTION
**Pull request by loosebazooka**
Just prepping for sigstore-java 1.0 (we haven't released yet).

### CODE REVIEW

File: content/announcing-sigstore-java-1-0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Signatures on Maven Central
+Thanks to the team at Sonatype, Sigstore signature bundles (`*.sigstore.json`) are an accepted signature format on Maven Central. So you can start signing your Maven Central artifacts with Sigstore today. To ensure existing verification workflows and tooling continue to work for Java workflows PGP signing is currently still required for publishing to Maven Central.
+
+## Features
+The `sigstore-java` library is meant to provide a comprehensive API to sigstore infranstucture. Giving users access to both simple high level API while also making it possible to tweak the behavior according to your needs.
+
+The 1.0 ships with these features today
+- Programmatic signing and verifying using sigstore-java APIs
+- Easy signing using Maven and Gradle Sigstore plugins
+- Signing identity via browser workflow or from Github Actions id-token
```
Comment (haydentherapper) [inline content/announcing-sigstore-java-1-0.md:25]: Does this support only GitHub Actions or other CI providers like GitLab?

Comment (loosebazooka) [reply content/announcing-sigstore-java-1-0.md:25]: Only github actions for now. We'll add in the other providers in 1.1.0

**Review by haydentherapper — DISMISSED** [review]

**Comment by haydentherapper** [comment]
TSC for review: @bobcallaway @priyawadhwa @lukehinds @SantiagoTorres @trevrosen

**Review by trevrosen — DISMISSED** [review]
Nice post - congrats on the ship! 🎉

File: content/announcing-sigstore-java-1-0.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Signatures on Maven Central
+Thanks to the team at Sonatype, Sigstore signature bundles (`*.sigstore.json`) are an accepted signature format on Maven Central. So you can start signing your Maven Central artifacts with Sigstore today. To ensure existing verification workflows and tooling continue to work for Java workflows PGP signing is currently still required for publishing to Maven Central.
+
+## Features
+The `sigstore-java` library is meant to provide a comprehensive API to sigstore infranstucture. Giving users access to both simple high level API while also making it possible to tweak the behavior according to your needs.
```
Comment (bobcallaway) [inline content/announcing-sigstore-java-1-0.md:20]:
```suggestion
The `sigstore-java` library is meant to provide a comprehensive API to sigstore infrastructure. It gives users access to both simple high level APIs while also making it possible to tweak the behavior according to your needs.
```

File: content/announcing-sigstore-java-1-0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The 1.0 ships with these features today
+- Programmatic signing and verifying using sigstore-java APIs
+- Easy signing using Maven and Gradle Sigstore plugins
+- Signing identity via browser workflow or from Github Actions id-token
+- Sigstore signature bundle support (`*.sigstore.json`)
+
+The client is already in use by some early adopters, and we attach sigstore signature bundles to all our sigstore-java artifacts on Maven Central. Head over to our github page ([sigstore-java](https://github.com/sigstore/sigstore-java), [maven](https://github.com/sigstore/sigstore-java/tree/main/sigstore-maven-plugin), [gradle](https://github.com/sigstore/sigstore-java/tree/main/sigstore-gradle)) for further details on accessing the API or integrating directly with your Java builds.
+
+## Development, Testing and Contributing
+Sigstore is an open source project developed directly on github and made possible by a dedicated and inclusive community.
```
Comment (bobcallaway) [inline content/announcing-sigstore-java-1-0.md:31]:
```suggestion
Sigstore is an open source project developed directly on GitHub and made possible by a dedicated and inclusive community.
```

File: content/announcing-sigstore-java-1-0.md (hunk 1 above)
Comment (hboutemy) [reply content/announcing-sigstore-java-1-0.md:25]: is there an issue to track that?

Comment (hboutemy) [reply content/announcing-sigstore-java-1-0.md:25]: I found https://github.com/sigstore/sigstore-java/issues/780 for Jenkins
I don't know if there are other ones for GitLab or others

Comment (loosebazooka) [reply content/announcing-sigstore-java-1-0.md:25]: Jenkins is a little harder to support as it's mostly self hosted, I don't know how we'd get the oidc provider into fulcio. I'll add an ambient bug here: https://github.com/sigstore/sigstore-java/issues/795

**Comment by bobcallaway** [comment]
LGTM (needs date and draft status fixed once release goes live)

**Comment by loosebazooka** [comment]
will update here when release is done

**Comment by loosebazooka** [comment]
Hrmm, looks like I need three reviews: anyone got a second to take another look? @bobcallaway @priyawadhwa @lukehinds @SantiagoTorres @trevrosen

**Review by bobcallaway — DISMISSED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Comment by loosebazooka** [comment]
release is done, thanks everyone

**Pull request merged**